task/fbc-validation/0.1/fbc-validation.yaml

```diff
@@ -39,7 +39,7 @@ spec:
 
         mkdir confdir
         if ! oc image extract $(params.IMAGE_URL) --file /bin/opm --file /bin/grpc_health_probe --path $conffolder*:confdir/ ; then
-        echo "Unable to extract image! Skipping checking binaries!"
+          echo "Unable to extract image! Skipping checking binaries!"
           HACBS_TEST_OUTPUT="$(make_result_json -r ERROR -f 1)"
```

Comment:
Should we add these error message to `note` of `HACBS_TEST_OUTPUT` then its readers can read it instead of check it in log

Reply:
We have limitation for length of the output as 4096B, right?

Reply:
Yes, I feel 4096B might be enough, but to be safe, let's just add the `note` for the `ERROR` result, not add to `FAILURE` result because there might be multiple failures?

Reply:
Good thinking, just add note for ERROR result. Including multiple failures could be like ticking bomb issue.